samples/bluetooth/peripheral_lbs/sample.yaml

```diff
@@ -5,30 +5,31 @@ tests:
   sample.bluetooth.peripheral_lbs:
     build_only: true
     integration_platforms:
-      - nrf52dk_nrf52832
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf5340dk_nrf5340_cpuapp_ns
-      - thingy53_nrf5340_cpuapp
-      - thingy53_nrf5340_cpuapp_ns
-      - nrf54l15pdk_nrf54l15_cpuapp
-    platform_allow: nrf52dk_nrf52832 nrf52840dk_nrf52840
-      nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuapp_ns thingy53_nrf5340_cpuapp
-      thingy53_nrf5340_cpuapp_ns nrf54l15pdk_nrf54l15_cpuapp
+      - nrf52dk/nrf52832
+      - nrf52840dk/nrf52840
+      - nrf5340dk/nrf5340/cpuapp
+      - nrf5340dk/nrf5340/cpuapp/ns
+      - thingy53/nrf5340/cpuapp
+      - thingy53/nrf5340/cpuapp/ns
+      - nrf54l15pdk/nrf54l15/cpuapp
+    platform_allow: nrf52dk/nrf52832 nrf52840dk/nrf52840
+      nrf5340dk/nrf5340/cpuapp nrf5340dk/nrf5340/cpuapp/ns thingy53/nrf5340/cpuapp
+      thingy53/nrf5340/cpuapp/ns nrf54l15pdk/nrf54l15/cpuapp
     tags: bluetooth ci_build
   sample.bluetooth.peripheral_lbs_minimal:
     build_only: true
     extra_args: OVERLAY_CONFIG=prj_minimal.conf
     integration_platforms:
-      - nrf52dk_nrf52810
-      - nrf52840dk_nrf52811
-      - nrf52833dk_nrf52820
-    platform_allow: nrf52dk_nrf52810 nrf52840dk_nrf52811 nrf52833dk_nrf52820
+      - nrf52dk/nrf52810
+      - nrf52840dk/nrf52811
+      - nrf52833dk/nrf52820
+    platform_allow: nrf52dk/nrf52810 nrf52840dk/nrf52811 nrf52833dk/nrf52820
     tags: bluetooth ci_build
   sample.bluetooth.peripheral_lbs_no_security:
     build_only: true
     extra_args: CONFIG_BT_LBS_SECURITY_ENABLED=n
     integration_platforms:
+<<<<<<< HEAD
```

Comment:
merge not completed,
found also in:
./samples/matter/light_bulb/sample.yaml:<<<<<<< HEAD
./samples/matter/light_switch/sample.yaml:<<<<<<< HEAD
./samples/bluetooth/peripheral_hids_mouse/sample.yaml:<<<<<<< HEAD
./samples/bluetooth/peripheral_lbs/sample.yaml:<<<<<<< HEAD

Reply:
fixed